src/components/forms/constructor-form/constructor-form.js

```diff
@@ -114,40 +123,46 @@ const ConstructorForm = ({ isEdit, editableConstructorElement }) => {
       default: editableConstructorElement.default || false,
       basePrice: +editableConstructorElement.basePrice[1].value / 100 || 0
     },
-
     onSubmit: () => {
       const constructorElement = createConstructor(values);
```

Comment:
You have `formValues `as a first value that `onSubmit` method accept.

Reply:
fixed